features/bootstrap/AccountHelper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Behatch\HttpCall\Request;
+
+class AccountHelper extends ResourceHelper
+{
+    public function __construct(Request $request)
+    {
+        parent::__construct($request);
+    }
+
+    public function createResource()
+    {
+        $body =<<<EOF
+    {
+      "username": "string",
+      "email": "string@string.fr",
+      "firstName": "string",
+      "lastName": "string",
+      "password": "password",
+      "salt": "salt"
+    }
+EOF;
+
+        $response = $this->request->send(
```

Comment:
maybe this part could be moved to the parent class and called passing two arguments (route and body)?